.env

```diff
@@ -53,4 +53,6 @@ ACCOUNT_LESS_STRICT_PASSWORD_RULES=FALSE
 MAILER_FROM_ADDR='email@domain.null'
 MAILER_FROM_NAME='SenderName'
 PELAGOS_NAME=pelagos
+ISSUE_TRACKING_BASE_URL='https://griidc.atlassian.net/issues'
+MDAPP_LOGFILE='/opt/pelagos/var/logs/mdapp.log'
```

Comment:
The folder logs doesn't exist anymore, it has been renamed to 'log' (new Symfony thing)

Reply:
BUT we can still use the same file path. 
Also you can use: `'%kernel.project_dir%/var/logs/...'`